plugins/rapid7_insight_agent/unit_test/test_quarantine_multiple.py

```diff
@@ -14,11 +14,8 @@
 @patch("requests.sessions.Session.post", side_effect=Util.mocked_request)
 class TestQuarantineMultiple(TestCase):
     @parameterized.expand(Util.load_json("parameters/quarantine_multiple_success.json.resp").get("parameters"))
-    @patch(
-        "icon_rapid7_insight_agent.util.graphql_api.api_connection.ApiConnection._get_agent_id",
-        return_value="assetID",
-    )
-    def test_quarantine_multiple(self, agent_id_array, interval, quarantine_state, expected, mock_agent_id, mock_post):
+    def test_quarantine_multiple(self, mock_post, agent_id_array, interval, quarantine_state, expected):
```

Comment:
Is it possible to also test failed array, and if we try/catch also if handled responses are correctly added to failed array?